pages/index.html

```diff
@@ -1,12 +1,57 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <title>Community & Human Services in King County</title>
+  <link href="/stylesheets/style.css" rel="stylesheet">
+  <link href="/stylesheets/index.css" rel="index_stylesheet">
 </head>
 <body>
-  
+  <header>
+      <h1>Community & Human Services in King County</h1>
+      <nav>
+          <ul>
+              <li><a href="facts.html">Facts</a></li>
+              <li><a href="gallery.html">Gallery</a></li>
+          </ul>
+      </nav>
+  </header>
+  <main>
+    <h2>Community Based Practices</h2>
+    <ul>
+        <li>
+            <section>
+                <h2>Ada's Mission </h2>
+                <img alt="Ada Logo" src=/images/ada_logo.jpg id="ada_logo" width="95">
+                <ul>
+                    <li>Ada’s mission is to prepare women and gender expansive adults to be software developers while advocating for inclusive and equitable work environments.<br></li> 
+                    <li>They primarily serve and address the needs of Black, Latine, Indigenous Americans, Native Hawaiian & Pacific Islander, LGBTQIA+, and low-income people.<br></li>
+                    <li>Their tuition-free training program includes a collaborative learning environment, individualized support (mentors, tutors, a network of mental health supports, affinity groups), and an applied learning internship.<br></li>
```

Comment:
`<br>` tags are not semantic HTML-- to achieve a line-break, I would expect the use of CSS. Same applies for the below `li`s